docs/virtual_machines/liveness_and_readiness_probes.md

```diff
@@ -77,95 +84,111 @@ VirtualMachineInstance, after an initial delay of 120 seconds. The
+          bus: virtio
         name: cloudinitdisk
+      rng: {}
+    resources:
+      requests:
+        memory: 1024M
+  livenessProbe:
+    initialDelaySeconds: 120
+    periodSeconds: 20
+    tcpSocket:
+      port: 1500
+    timeoutSeconds: 10
+  terminationGracePeriodSeconds: 0
+  volumes:
+  - name: containerdisk
+    containerDisk:
+      image: quay.io/containerdisks/fedora
+  - cloudInitNoCloud:
+      userData: |-
+        #cloud-config
+        password: fedora
+        user: fedora
+        chpasswd: { expire: False }
+        bootcmd:
+          - ["sudo", "dnf", "install", "-y", "nmap-ncat"]
+          - ["sudo", "systemd-run", "--unit=httpserver", "nc", "-klp", "1500", "-e", '/usr/bin/echo -e HTTP/1.1 200 OK\\nContent-Length: 12\\n\\nHello World!']
+    name: cloudinitdisk
+```
 
 ## Define Readiness Probes
 
 Readiness Probes are configured in a similar way like liveness probes.
 Instead of `spec.livenessProbe`, `spec.readinessProbe` needs to be
 filled:
 
-    apiVersion: kubevirt.io/v1alpha3
-    kind: VirtualMachineInstance
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    kubevirt.io/vm: vmi-fedora-vmi
+  name: vmi-fedora
+spec:
+  template:
     metadata:
       labels:
-        special: vmi-fedora
-      name: vmi-fedora
-    spec:
-      domain:
-        devices:
-          disks:
-          - disk:
-              bus: virtio
-            name: containerdisk
-          - disk:
-              bus: virtio
-            name: cloudinitdisk
-        resources:
-          requests:
-            memory: 1024M
-      readinessProbe:
-        httpGet:
-          port: 1500
-        initialDelaySeconds: 120
-        periodSeconds: 20
-        timeoutSeconds: 10
-        failureThreshold: 3
-        successThreshold: 3
-      terminationGracePeriodSeconds: 0
-      volumes:
-      - name: containerdisk
-        registryDisk:
-          image: registry:5000/kubevirt/fedora-cloud-registry-disk-demo:devel
-      - cloudInitNoCloud:
-          userData: |-
-            #cloud-config
-            password: fedora
-            chpasswd: { expire: False }
-            bootcmd:
-              - setenforce 0
-              - dnf install -y nmap-ncat
-              - systemd-run --unit=httpserver nc -klp 1500 -e '/usr/bin/echo -e HTTP/1.1 200 OK\\n\\nHello World!'
+        kubevirt.io/domain: vmi-fedora
+        kubevirt.io/vm: vmi-fedora
+  domain:
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
         name: cloudinitdisk
+      rng: {}
+    resources:
+      requests:
+        memory: 1024M
+  readinessProbe:
+    initialDelaySeconds: 120
+    periodSeconds: 20
+    timeoutSeconds: 10
+    failureThreshold: 3
+    successThreshold: 3
+    httpGet:
+      port: 1500
+  terminationGracePeriodSeconds: 0
+  volumes:
+  - name: containerdisk
+    containerDisk:
+      image: quay.io/containerdisks/fedora
```

Comment:
Same as above.

Reply:
All done! Thanks for your suggestions :)